src/components/Dashboard.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react';
+import './Dashboard.css'
+import DashboardNavBar from './DashboardNavbar'
+import ActionScreen from './ActionScreen'
+import { BrowserRouter as Router } from 'react-router-dom';
+
+const Dashboard = (props) => {
+
+    const [pokemons, setPokemons] = useState({})
+
+    useEffect(() => {
+        fetch('https://pokeapi.co/api/v2/pokemon?limit=11')
+        .then(response => response.json())
+        .then(pokemons => extractData(pokemons))
+
+        const extractData = (pokemons) => {
+            pokemons.results.forEach(pokemon => {
+                const url = pokemon.url
+                fetch(url)
+                .then(response => response.json())
+                .then(pokemons => setPokemons(pokemons))
+            });
+        }
+    })
```

Comment:
N+1 sequential API requests

Reply:
Fetch in parallel. Look into using Promise.all.

Reply:
To run this useEffect only once - useEffect(() => {}, []);

Otherwise this is an infinite loop